content/en/tracing/trace_collection/admission_controller.md

```diff
@@ -15,6 +15,8 @@ First, configure the Datadog Agent. Our recommended method of installation for K
 ## Unified Service Tagging
 For your Kubernetes applications that send traces to Datadog Agent, you can configure the Datadog admission controller to inject Java, JavaScript, and Python libraries automatically. To inject the Java and Javascript libraries, use Datadog Cluster Agent v7.39+. To inject the Python library, use Datadog Cluster Agent v7.40+.
```

Comment:
Side question, since there are differences per language, would it be helpful to have separate tabs about what is required for Java, JS, and Python to work?